<launch>
    <arg name="visualization" default="true"/>
    <arg name="location" default="sertig"/>

    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>

    <node pkg="terrain_planner" type="test_mcts_mapping" name="mcts_planner" output="screen">
        <param name="terrain_path" value="$(find terrain_planner)/../resources/$(arg location).tif"/>
        <param name="terrain_color_path" value="$(find terrain_planner)/../resources/$(arg location)_color.tif"/>
    </node>
    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find terrain_planner)/launch/config_mcts_mapping.rviz" />
    </group>
</launch>
